package.json

```
@@ -6,7 +6,7 @@
     "titanium",
     "mobile"
   ],
-  "version": "5.1.9",
+  "version": "5.2.0",
```

Let's do a major release.

```suggestion
  "version": "6.0.0",
```

While you're updating the `package.json`, could you please set the `"author": "TiDev, Inc. <npm@tidev.io>"` and remove the `maintainers`.

We really should bump the min Node.js version to Node 16. Even then, we should drop Node 16 and 18. Ugh, maybe a larger effort. Dunno.

---

sure, done :+1: I think if we do a major update we should merge https://github.com/tidev/node-titanium-sdk/pull/652 too so the Arm Mac users are happy again :smile:

---

Ok, I'll merge this and wait to publish until #652 has been reviewed/merged.